File: app/src/main/java/com/mailerdaemon/app/LostAndFound/LostAndFoundModel.java
package com.mailerdaemon.app.LostAndFound;

import com.google.firebase.firestore.ServerTimestamp;

import java.util.Date;

import lombok.Data;

@Data
public class LostAndFoundModel {
    private String heading;
    private String details;
    private String photo;
    @ServerTimestamp
    private Date date;
    private Boolean verified;
    private String uid;
}

File: app/src/main/java/com/mailerdaemon/app/lostAndFound/LostAndFoundModel.kt
package com.mailerdaemon.app.lostAndFound

import com.google.firebase.firestore.ServerTimestamp
import java.util.Date

data class LostAndFoundModel(
    var heading: String? = null,
    var details: String? = null,
    var photo: String? = null,
    @ServerTimestamp
    var date: Date? = null,
    var verified: Boolean? = null,
    var uid: String? = null
)
